DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/CMakeLists.txt

```diff
@@ -0,0 +1,108 @@
+# To see a Makefile equivalent of this build system:
+# https://github.com/oneapi-src/oneAPI-samples/blob/master/DirectProgramming/DPC++/ProjectTemplates/makefile-fpga
+
+set(SOURCE_FILE merge_sort.cpp)
+set(TARGET_NAME merge_sort)
+set(EMULATOR_TARGET ${TARGET_NAME}.fpga_emu)
+set(SIMULATOR_TARGET ${TARGET_NAME}.fpga_sim)
+set(FPGA_TARGET ${TARGET_NAME}.fpga)
+
+# FPGA board selection
+if(NOT DEFINED FPGA_BOARD)
+    set(FPGA_BOARD "intel_a10gx_pac:pac_a10")
+    message(STATUS "FPGA_BOARD was not specified.\
+                    \nConfiguring the design to run on the default FPGA board ${FPGA_BOARD} (Intel(R) PAC with Intel Arria(R) 10 GX FPGA). \
+                    \nPlease refer to the README for information on board selection.")
+else()
+    message(STATUS "Configuring the design to run on FPGA board ${FPGA_BOARD}")
+endif()
+
+# check if the BSP has USM host allocations or manually enable using host allocations
+# e.g. cmake .. -DUSE_USM_HOST_ALLOCATIONS=1
+if(FPGA_BOARD MATCHES ".usm.*" OR DEFINED USE_USM_HOST_ALLOCATIONS)
+    set(ENABLE_USM "-DUSM_HOST_ALLOCATIONS")
+    message(STATUS "USM host allocations are enabled")
+endif()
+
+# This is a Windows-specific flag that enables error handling in host code
+if(WIN32)
+    set(WIN_FLAG "-EHsc")
```

Comment:
/EHsc

Reply:
I thought Alain was changing all these...?

Reply:
Yeah, that was a mistake (by me), we don't actually want to change these, they should still be /